boards/arm/stm32l476g_disco/doc/stm32l476g_disco.rst

```diff
@@ -0,0 +1,231 @@
+.. _stm32l476g_disco_board:
+
+ST STM32L476G Discovery
+########################
+
+Overview
+********
+
+The STM32L476G Discovery board features an ARM Cortex-M4 based STM32L476VG MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the STM32L476G Discovery board:
+
+
+- STM32L476VGT6 microcontroller featuring 1 Mbyte of Flash memory, 128 Kbytes of RAM in LQFP100 package
+- On-board ST-LINK/V2-1 supporting USB re-enumeration capability
+- Three different interfaces supported on USB:
+
+        - Virtual com port
```

Comment:
When rendered as html, these indented lists (here and below) look better when they're not indented so much.  Instead of 8 spaces use 4.